main.css

```diff
@@ -544,3 +544,252 @@ button:hover {
+    width: 650px;
+    height: 146px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    margin: 0.5px;
+  }
+
+  .projects {
+    display: none;
+  }
+
+  .projects-web {
+    display: flex;
+    align-items: center;
+  }
+
+  .project-card {
+    display: flex;
+  }
+
+  .proj-webcad {
+    display: flex;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    padding: 24px;
+    margin: 5%;
+  }
+
+  .web-proj {
+    display: flex;
+    flex-direction: column;
+    margin: 20px;
+    align-items: center;
+  }
+
+  li.skillset {
+    font-size: 12px;
+    letter-spacing: 0.03em;
+  }
+
+  .project-card2 {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .abt-sec {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 10px;
+  }
+
+  .abt-section {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 10px;
+  }
+
+  .section-about {
+    display: flex;
+    margin: 30px;
+  }
+
+  .contact-me {
+    background-color: #6070ff;
+    border: 1px solid #cfd8dc;
+    border-radius: 100px 0 0;
+    width: 100%;
+  }
+
+  .bg-image {
+    background-image: url(images/contact-form-background-shapes.png);
+    background-repeat: no-repeat;
+    background-position-x: right;
+  }
+
+  .contact-title {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    height: 56px;
+    justify-content: center;
+    padding-top: 80px;
+  }
+
+  .contact-text {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #ebebff;
+    justify-content: center;
+    height: 146px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    margin: auto;
+    width: 569px;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin: 0 8px;
+  }
+
+  #username {
+    background: #fff;
+    height: 48px;
+    width: 327px;
+    left: 16px;
+    right: 38px;
+    margin: 5px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding-left: 16px;
+    font-size: 18px;
+    color: #172b4d;
+  }
+
+  #useremail {
+    background: #fff;
+    height: 48px;
+    width: 327px;
+    left: 16px;
+    right: 38px;
+    margin: 5px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding-left: 16px;
+    font-size: 18px;
+    color: #172b4d;
+  }
+
+  #usermessage {
+    height: 170px;
+    width: 327px;
+    left: 24px;
+    top: 5125px;
+    margin: 5px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding-top: 15px;
+    padding-left: 16px;
+    font-size: 15px;
+    color: #b3bac5;
+  }
+}
```

Comment:
### Please stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible
- [ ] [REQUIRED] Kindly note that your desktop version of the project still has horizontal and vertical scroll bars. The previous reviewer also pointed this out. Please make the width 100% and don't fix it so your design can fit different desktop screen sizes in order to avoid the scroll bars. 
Also, kindly check all your cards on the desktop version, the container that holds the card seem to be smaller as it does not contain all the contents of the card. Please fix the sizes of language buttons as it looks bigger than the ones in the figma template. It will be nice to also check the alignment of your text as well. 
### Your design
![scroll-bar](https://user-images.githubusercontent.com/58692590/149368039-3129a319-a23e-49aa-996d-558448237a78.png)
### Figma Template
![Screenshot from 2022-01-13 17-23-38](https://user-images.githubusercontent.com/58692590/149368917-6023821c-01ea-4c09-b92f-3926e6e3044d.png)